### ISSUE TITLE
Analyse en 1 fois d'une solution Etuis/Fardeau/caisse/palette

### ISSUE THREAD

**Issue by fdicresc** [treeDiM/StackBuilder#115]
Bonjour,

J'aimerai pouvoir avoir au sein d'un même rapport, l'analyse me permettant de choisir la meilleur optimisation en partant de l'étui, avec la création de fadeau à ranger dans une caisse et ensuite la paletisé.
Aujourd'hui, je suis obligé de créer un étui, de faire une première analyse étui caisse qui va me permettre de trouver un fardelage optimal (manuel), ensuite je créée le fardeau, réalise une analyse fardeau/caisse et enfin je doit recréer une analyse récupérant la caisse précédente pour le mettre sur une palette.

L'idée étant de pouvoir regrouper au maximum pour que le rapport puisse être le plus exaustif possible et proposé en une fois les informations sur l'étui, le fardeau, la caisse et la palette.

Merci de votre retour.
Bonne soirée

**Comment by fgasnier71** [comment]
Bonjour,
Quelle version du logiciel utilisez-vous?
Si je comprend bien votre demande, la version 3.0 permet d'effectuer cette tâche et d'obtenir un seul rapport.
Cordialement,

**Comment by fdicresc** [comment]
Bonsoir,
Je suis sur la 3.0.
Ma demande est de pouvoir le faire en une seule fois.
Aujourd'hui, je décompose mes actions ainsi :
* Analyse étui/caisse, ce qui me permet de déterminer de façon manuel le meilleur fardeau
* Création du fardeau
* Analyse Fardeau/caisse
* Analyse Caisse du résultat précédent/Palette

Je suis alors obligé de retravailler les rapports, les fusionner, pour avoir l'ensemble des éléments.
Si on pouvais faire une analyse Etui/Palette, en mentionnant les conditionnement intermédiaire, se serai très pratique.

Merci de votre aide.
Cordialement

**Comment by fgasnier71** [comment]
Je ne suis pas sûr de comprendre:
- vous encaissez des fardeaux?
- Un rapport unique ne vous permet pas d'obtenir toutes les données étui<fardeau<caisse<palette ?
Avez-vous constaté qu'on peut par exemple réutiliser le résultat d'une analyse étui/caisse comme donnée d'entrée d'une analyse caisse/palette?
Peut-être pourriez vous nous faire parvenir un exemple de rapport "fusionné" montrant le résultat souhaité?
Cordialment,

**Comment by fdicresc** [comment]
Oui, pour des faciliter de manutention, je regroupe mes étuis en fardeaux que je met en caisse.
Mon fardeau est le regroupement de 10 étuis par une bande de polyéthylène...
Non, car je ne peut pas utiliser l'ananlyse étuis/caisse, car cette dernière ne va pas optimiser par rapport au nombre de fardeaux que j'aurai dans la caisse.
J'ai vu pour la réutilisation des résultats, c'est ce que je fais pour l'nanlyse caisse /palette.
Il me manque juste l'étui et le fait que l'analyse puisse me donner le meilleur fardeau pour ma caisse.
Ci-joint un rapport, surligneur en jaune, ce que j'ai ajouté et en bleu ce que j'ai déplacé pour respecter une présentation des conditionnements du plus petits au plus gros.
Cordialement
[Analyse2.docx](https://github.com/treeDiM/StackBuilder/files/1385092/Analyse2.docx)
